downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -43,7 +44,9 @@ class Git : GitBase() {
     override val priority: Int = 100
 
     override fun isApplicableUrlInternal(vcsUrl: String) =
-        ProcessCapture("git", "ls-remote", vcsUrl).isSuccess
+        suppressInput {
+            ProcessCapture("git", "-c", "credential.helper=", "-c", "core.askpass=", "ls-remote", vcsUrl).isSuccess
```

Comment:
Did you confirm that this helps for non Windows systems? If so, could you pleae explain in the commit message. Also, there's a typo in "unfortunately".

Reply:
Yes, I did. I've reworded the commit message.